astropy/io/fits/fitstime.py

```diff
@@ -550,20 +550,17 @@ def time_to_fits(table):
         # By default, Time objects are written in full precision, i.e. we store both
         # jd1 and jd2 (serialize_method['fits'] = 'jd1_jd2'). Formatted values for
         # Time can be stored if the user explicitly chooses to do so.
+        col_cls = MaskedColumn if col.masked else Column
```

Comment:
I'm not sure that when the code was written `Time` could be masked - but since masking uses `np.nan` it does not matter for `jd1_jd2`. But it does matter for 'formatted_value' for string-like formats that the mask is kept.

Reply:
At the time of writing this code, I don't think I explicitly dealt with masked values -- I assume that `Time` could not be masked then. I agree that for the default full precision `jd1_jd2` `np.nan` will do the job.